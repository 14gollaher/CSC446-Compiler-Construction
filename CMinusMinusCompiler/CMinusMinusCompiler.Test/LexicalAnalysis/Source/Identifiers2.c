abc!!!
123abc
_abc
abc?def
abc def
