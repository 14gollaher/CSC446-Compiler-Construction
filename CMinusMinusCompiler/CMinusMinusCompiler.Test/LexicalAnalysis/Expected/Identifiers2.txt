Lexeme                      Token                Attributes     
-----------------------------------------------------------
ERROR: Line 1 - Unexpected character in identifier
ERROR: Line 2 - Unexpected character in identifier
ERROR: Line 3 - Unexpected character in identifier
ERROR: Line 4 - Unexpected character in identifier
ERROR: Line 3 - Identifier length exceeds maximum 27 characters
abc                         IdentifierToken
def                         IdentifierToken
\n                          EndOfFileToken